posts/2023-08-21_date_functions_and_imputation/date_functions_and_imputation.qmd

```diff
@@ -0,0 +1,252 @@
+---
+title: "Date/Time Functions and Imputation in {admiral} "
+author:
+  - name: Edoardo Mancini
+description: "Dates, times and imputation can be a frustrating facet of any programming language. Here's how {admiral} makes all of this easy!"
+date: "2023-08-21"
+# please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [admiral]
```

Comment:
Lets add ADaM and mabye create a new tag for Date/Time?  We can the Date/Time to the other blog posts.